src/js/Setup.js

```diff
@@ -356,6 +356,61 @@ function createCindyNow() {
     if (typeof scriptconf === "string" && scriptconf.search(/\*/) >= 0) scriptpat = scriptconf;
     if (typeof scriptconf !== "object") scriptconf = null;
 
+    /*
+Loads CindyScript files (marked by .cjs file ending) and adds their code AT THE BEGINNING of the init script.
+The file names (and paths) have to be listed in the dictionary that's passed to CindyJS/createCindy as an array for the key 'import'. I.e., like this:
+          var cdy = CindyJS({
+                ports: [{ id: "CSCanvas", width: 500, height: 500 }],
+                scripts: "cs*",
+                import: ["cindyscript_libraries/libraryA", "cindyscript_libraries/libraryB"]
+
+            });
+Note that the libraries get prepended to the init-script BACK TO FRONT. So, in the example above, the code from libraryA comes before the code from libraryB whoch comes before the custom user code. That way, libraries can reference each others code.
+
+CAUTION!
+Since this uses the 'fetch' command, it only works on a web server. So, for local testing, start one with 'python -m http.server' or however else you are comfortable with.
```

Comment:
It would be nice to fall back to regular file loading if this is not on a webserver.

Reply:
And actually I wonder whether this could be done using a `file://` URL, but I do not have detailed knowledge about this in JavaScript, so this is a wild guess.

Reply:
Yes, it would!
I don't know how to check for that, yet. But I will look into it now...

Reply:
Ok, I will wait a little with merging then.

Reply:
Alright, as far as I can see, CindyJS proceeds as normal if the fetch command fails due to not being on a web server. (Of course, missing code from the not loaded code files might be an issue.) But I added a console output, that makes it clearer that this is the case.